deployment/src/main/java/io/quarkiverse/fury/deployment/FurySerializerBuildItem.java

```diff
@@ -30,16 +36,24 @@ public Class<? extends Serializer> getSerializer() {
         return serializer;
     }
 
-    public static FurySerializerBuildItem buildItem(ClassInfo classInfo) {
+    public static List<FurySerializerBuildItem> buildItems(ClassInfo classInfo) {
         Class<?> clazz = JandexReflection.loadClass(classInfo);
         FurySerialization annotation = clazz.getDeclaredAnnotation(FurySerialization.class);
-        if (annotation.targetClass() != FurySerialization.CurrentAnnotatedTypeStub.class) {
-            clazz = annotation.targetClass();
-        }
+        Class<?>[] classes = annotation.targetClasses();
+        int classed = annotation.classId();
```

Comment:
`classed` is a typo? I guess it is `classId`?

Reply:
yeah, it's a typo, thanks fo pointing this out